contrib/debian/copyright

```diff
@@ -23,60 +23,50 @@ Files: debian/manpages/*
-       src/qt/res/src/*.svg
-Copyright: Wladimir van der Laan
-License: Expat
-
-Files: src/qt/res/icons/address-book.png, src/qt/res/icons/export.png,
-       src/qt/res/icons/history.png, src/qt/res/icons/key.png,
-       src/qt/res/icons/lock_*.png, src/qt/res/icons/overview.png,
-       src/qt/res/icons/receive.png, src/qt/res/icons/send.png,
-       src/qt/res/icons/synced.png, src/qt/res/icons/filesave.png
-Copyright: David Vignoni (david@icon-king.com)
-           ICON KING - www.icon-king.com
-License: LGPL
-Comment: NUVOLA ICON THEME for KDE 3.x
-         Original icons: kaddressbook, klipper_dock, view-list-text,
-                         key-password, encrypted/decrypted, go-home, go-down,
-                         go-next, dialog-ok
-         Site: http://www.icon-king.com/projects/nuvola/
-
-Files: src/qt/res/icons/connect*.png
-Copyright: schollidesign
-License: GPL-3+
-Comment: Icon Pack: Human-O2
-         Site: http://findicons.com/icon/93743/blocks_gnome_netstatus_0
-
-Files: src/qt/res/icons/transaction*.png
-Copyright: md2k7
-License: Expat
-Comment: Site: https://bitcointalk.org/index.php?topic=15276.0
-
-Files: src/qt/res/icons/configure.png, src/qt/res/icons/quit.png,
-      src/qt/res/icons/editcopy.png, src/qt/res/icons/editpaste.png,
-      src/qt/res/icons/add.png, src/qt/res/icons/edit.png,
-      src/qt/res/icons/remove.png
-Copyright: http://www.everaldo.com
-License: LGPL
-Comment: Icon Pack: Crystal SVG
+Files: src/qt/res/icons/add.png,
+       src/qt/res/icons/address-book.png,
+       src/qt/res/icons/configure.png,
+       src/qt/res/icons/debugwindow.png,
+       src/qt/res/icons/edit.png,
+       src/qt/res/icons/editcopy.png,
+       src/qt/res/icons/editpaste.png,
+       src/qt/res/icons/export.png,
+       src/qt/res/icons/eye.png,
+       src/qt/res/icons/filesave.png,
+       src/qt/res/icons/history.png,
+       src/qt/res/icons/info.png,
+       src/qt/res/icons/key.png,
+       src/qt/res/icons/lock_*.png,
+       src/qt/res/icons/open.png,
+       src/qt/res/icons/overview.png,
+       src/qt/res/icons/quit.png,
+       src/qt/res/icons/receive.png,
+       src/qt/res/icons/remove.png,
+       src/qt/res/icons/send.png,
+       src/qt/res/icons/synced.png,
+       src/qt/res/icons/transaction*.png,
+       src/qt/res/icons/tx_output.png,
+       src/qt/res/icons/warning.png
+Copyright: Stephen Hutchings (and more)
+           http://typicons.com
+License: MIT
+Comment: Site: https://github.com/stephenhutchings/typicons.font
+
+Files: src/qt/res/icons/connect*.png, src/qt/res/src/connect-*.svg
+Copyright: Marco Falke
+License: MIT
+Comment: Inspired by Stephan Hutchings Typicons
+
+Files: src/qt/res/icons/tx_mined.png
+Copyright: Jonas Schnelli
+License: MIT
+Comment:
 
-Files: src/qt/res/icons/bitcoin.png, src/qt/res/icons/toolbar.png
-Copyright: Bitboy (optimized for 16x16 by Wladimir van der Laan)
+Files: src/qt/res/icons/bitcoin.*, src/qt/res/src/bitcoin.svg
+Copyright: Bitboy
```

Comment:
Thanks for letting me know. I fixed that and created another PR. There will be more noise but I think it is easier to discuss or merge when the PRs are split.